deployment/helmfiles/dev/helmfile.yaml

```diff
@@ -1,20 +1,32 @@
 repositories:
   - name: bitnami
     url: https://charts.bitnami.com/bitnami
+  - name: common
+    url: https://pjgg.github.io/helm-charts-parent/
```

Comment:
I guess this URL needs to be changed to http://quarkus-qe.github.io/helm-charts-parent , right?